src/scripts/components/VideoListItem.js

```diff
@@ -132,20 +132,24 @@ class VideoListItem extends Component<Props, void> {
       return <ListItem>Something went wrong - no video known</ListItem>
     }
 
+    let statusMessage = ''
+
     if (
-      video.storageStatus.name === 'success' &&
+      video.uploadStatus.name === 'success' &&
       video.transcodingStatus.name === 'success'
     ) {
-      isReady = true
-    }
-
-    let statusMessage = ''
-    if (video.storageStatus.name !== 'success' && title === null) {
-      statusMessage = 'Please provide a title and description'
-    } else if (video.transcodingStatus.name === 'success') {
-      statusMessage = 'Your video is ready to play'
-    } else if (video.transcodingStatus.name === 'failed') {
-      statusMessage = 'Your video could not be transcoded'
+      if (video.title.length < 1) {
```

Comment:
maybe you could also include the `storageStatus` in the if 
`if (video.title.length < 1 && video.storageStatus.name !== 'success') {`

Reply:
@eliawk the `video.storageStatus.name` always returns `idle`. And the notifications take care ok those statuses.

I'm gonna take care of the errors https://github.com/Paratii-Video/paratii-portal/issues/387